wavefront/wavefront.go

```diff
@@ -181,8 +235,10 @@ func writeMetrics(cfg *config.Params, insts []*metric.InstanceMsg) {
 			if float64Val, err := translateToFloat64(value); err != nil {
 				log.Warnf("couldn't translate metric value: %s %v, err: %v", metricName, value, err)
 			} else {
-				gauge := wf.GetOrRegisterMetric(metricName, metrics.NewGaugeFloat64(), tags).(metrics.GaugeFloat64)
+				gauge := metrics.NewGaugeFloat64()
```

Comment:
Should we use https://github.com/wavefrontHQ/go-metrics-wavefront/blob/3fca929deab620bfd77b9749a5f0cf221c10e62b/reporting/reporter.go#L364 here ?

Reply:
changes done as per comment.